main.js

```diff
@@ -200,6 +206,19 @@ export class IdEncoder {
    * @returns {string} - Encoded string.
    */
   encode(bytes) {
+    if(this.multihash) {
+      const BYTE_SIZE = bytes.length;
+      // <varint hash fn code> <varint digest size in bytes> <hash fn output>
+      //  <identity function>             <byte size>                <raw bytes>
+      const multihash = new Uint8Array(2 + BYTE_SIZE);
+      // <varint hash fn code>: identity function
+      multihash.set([MULTIHASH_IDENTITY_FUNCTION_CODE]);
+      // <varint digest size in bytes>
+      multihash.set([BYTE_SIZE], 1);
```

Comment:
Should check that the size is <= 127 (for simplicity for now). Otherwise this needs to be varint encoded.

Reply:
Added check 7aede2e